src/app/component/general/eco-news/eco-news-detail/eco-news-detail.component.ts

```diff
@@ -1,15 +1,38 @@
-import { Component, OnInit } from '@angular/core';
```

Comment:
please, fix errors in the console

Reply:
![image](https://user-images.githubusercontent.com/54113846/77179051-a0b24180-6ad0-11ea-9414-b6a80c7bc827.png)

Reply:
interpolation is incorrect

Reply:
date format is invalid
Please use date pipe

Reply:
I agreed with Roman, that I will fix issue with interpolation, as we need to change model file.

Reply:
I agreed with Roman, that I will fix issue with interpolation, as Model file have to be changed according to new JSON file.

Reply:
For Date pipe use same example of format date as on the Figma mockup. We agreed this today with BA and Designer.

Reply:
> For Date pipe use same example, of format date, as on the Figma mockup. We agreed this today with BA and Designer.

I will add pipe in my html

Reply:
> date format is invalid
> Please use date pipe('longDate')

It is better to use mediumDate due to mockup

Reply:
use simple date, as on Mockup no time only date

Reply:
> please, fix errors in the console

fixed